docs/reference/io.rst

```diff
@@ -8,16 +8,11 @@ Unified File Readers
 
 .. automodapi:: sunpy.io
 
-.. _iojp2:
-.. automodapi:: sunpy.io.jp2
-
-.. _ioana:
 .. automodapi:: sunpy.io.ana
 
 Special File Readers
 ====================
 
-.. _iospecialgenx:
```

Comment:
In general we shouldn't really remove these anchors, other sphinx projects outside of sunpy could be referencing them.

Reply:
I understand but if someone is linking to these, we have some problems.

There I think is a need to re-anchor large parts of our API pages in the future.